src/spaceObjects/playerSpaceship.cpp

```diff
@@ -1124,7 +1124,7 @@ bool PlayerSpaceship::hailCommsByGM(string target_name)
         return false;
 
     // Log the hail.
-    addToShipLog("Hailed by " + target_name, colorConfig.log_generic);
+    addToShipLog(tr("shiplog", "Hailed by ") + target_name, colorConfig.log_generic);
```

Comment:
This is one of the cases where it needs more changes then a simple `tr()`
`tr("shiplog", "Hailed by {name}").format({{"name", target_name}})`